### ISSUE TITLE
Transform object keys produce invalid code

### ISSUE THREAD

**Issue by olOwOlo** [javascript-obfuscator/javascript-obfuscator#569]
Obfuscate the following source code

```js
function renameKeys(obj, km) {
    const keyValues = Object.keys(obj).map(key => {
        const newKey = km[key] || key;
        return { [newKey]: obj[key] };
    });
    return Object.assign({}, ...keyValues);
}
```

by cli command line

```bash
javascript-obfuscator lib/utils/maps.js --output lib/utils/maps.bin.js --compact false --transform-object-keys true
```

## Expected Behavior

```diff
function renameKeys(_0x11a49e, _0x241472) {
    const _0x2b0dd8 = Object['keys'](_0x11a49e)[a0_0x5935('0x1')](_0x41462c => {
        const _0x4d0e55 = _0x241472[_0x41462c] || _0x41462c;
        const _0x248236 = {};
+       _0x248236[_0x4d0e55] = _0x11a49e[_0x41462c];
        return _0x248236;
    });
    return Object[a0_0x5935('0x0')]({}, ..._0x2b0dd8);
}
```

## Current Behavior

```diff
function renameKeys(_0x11a49e, _0x241472) {
    const _0x2b0dd8 = Object['keys'](_0x11a49e)[a0_0x5935('0x1')](_0x41462c => {
        const _0x4d0e55 = _0x241472[_0x41462c] || _0x41462c;
        const _0x248236 = {};
+       _0x248236[newKey] = _0x11a49e[_0x41462c];
        return _0x248236;
    });
    return Object[a0_0x5935('0x0')]({}, ..._0x2b0dd8);
}
```

## Your Environment

* Obfuscator version used: 0.25.2_2020-02-26T16:13:34.409Z
* Node version used: v13.8.0

---

Full output

```js
const a0_0x6cf7 = [
    'assign',
    'map'
];
(function (_0x38226d, _0x6cf733) {
    const _0x593543 = function (_0x55406b) {
        while (--_0x55406b) {
            _0x38226d['push'](_0x38226d['shift']());
        }
    };
    _0x593543(++_0x6cf733);
}(a0_0x6cf7, 0x150));
const a0_0x5935 = function (_0x38226d, _0x6cf733) {
    _0x38226d = _0x38226d - 0x0;
    let _0x593543 = a0_0x6cf7[_0x38226d];
    return _0x593543;
};
function renameKeys(_0x11a49e, _0x241472) {
    const _0x2b0dd8 = Object['keys'](_0x11a49e)[a0_0x5935('0x1')](_0x41462c => {
        const _0x4d0e55 = _0x241472[_0x41462c] || _0x41462c;
        const _0x248236 = {};
        _0x248236[newKey] = _0x11a49e[_0x41462c];
        return _0x248236;
    });
    return Object[a0_0x5935('0x0')]({}, ..._0x2b0dd8);
}
```

**Comment by sanex3339** [comment]
PR with the fix

**Comment by sanex3339** [comment]
Released as `0.25.4`

**Comment by olOwOlo** [comment]
Works now. Many thanks.
